clarifai_grpc/channel/clarifai_channel.py

```diff
@@ -48,20 +48,21 @@ def get_grpc_channel(base=None):
         wrap_response_deserializer = _response_deserializer_for_grpc
 
         if not base:
-            base = os.environ.get("CLARIFAI_GRPC_BASE")
-        if not base:
-            base = "api.clarifai.com"
+            base = os.environ.get("CLARIFAI_GRPC_BASE", "api.clarifai.com")
 
         return service_pb2_grpc.grpc.secure_channel(
             base, service_pb2_grpc.grpc.ssl_channel_credentials()
         )
 
     @staticmethod
-    def get_insecure_grpc_channel():
+    def get_insecure_grpc_channel(base=None, port=None):
         global wrap_response_deserializer
         wrap_response_deserializer = _response_deserializer_for_grpc
-        channel_address = "{}:18080".format(
-            os.environ.get("CLARIFAI_GRPC_BASE", "api-grpc.clarifai.com")
-        )
+
+        if not base:
+            base = os.environ.get("CLARIFAI_GRPC_BASE", "api-grpc.clarifai.com")
+        if not port:
+            port = "18080"
```

Comment:
what if the caller wants to use no port? they may set `base` param and will keep `port` set to None, but port is automatically changed to 18080.
instead of having two params, i suggest to have only one parameter `channel_address` . If it's set, then use it, otherwise, generate it from defaults.

Reply:
I'd prefer to keep `base` for similarity with `get_grpc_channel`. 
Also for backward compatibility, I still need to get `base` from env and use the default port, so having different ways to get args via env and function parameters will make code more complicated.

To address the problem with port==none, I've added the default value to the function signature.

I removed the default base, since we do not provide any public unsecured services.  This method is for clients who have on-prem installations.